homeassistant/components/mqtt_eventstream.py

```diff
@@ -44,6 +45,9 @@ def _event_publisher(event):
             return
         if event.event_type == EVENT_TIME_CHANGED:
             return
+        # filter out event from mqtt component about received messages
+        if event.event_type == EVENT_MQTT_MESSAGE_RECEIVED:
```

Comment:
This will prevent any MQTT messages from being bridged. Instead we should filter out received messages that we published to `pub_topic`